docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>"And the raw count data is:" -&gt; "And the raw data can be converted to a bitstring-count format:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/#comment-1690622936'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>